File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Microtest/Models/Prescriptions/PrescriptionCourse.cs
using System;

namespace NHSOnline.Backend.GpSystems.Suppliers.Microtest.Models.Prescriptions
{
    public class PrescriptionCourse
    {
        public string Id { get; set; }

        public DateTimeOffset? OrderDate { get; set; }

        public string Status { get; set; }
        
        public string Name { get; set; }

        public string Quantity { get; set; }

        public string Dosage { get; set; }

        public string Type { get; set; }

        public string Reason { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/microtest/prescriptions/PrescriptionCourse.kt
package mocking.microtest.prescriptions

data class PrescriptionCourse(var id: String,
                              var orderDate: String,
                              var status: PrescriptionStatus,
                              var name: String,
                              var quantity: String,
                              var dosage: String,
                              var type: String,
                              var reason: String
                              )
